docker/scripts/mininet/multi_switch_mininet.py

```diff
@@ -79,9 +81,11 @@ def main():
     conf = manifest['targets'][args.target]
     params = conf['parameters'] if 'parameters' in conf else {}
 
+    os.environ.update(dict(map(lambda (k,v): (k, str(v)), params.iteritems())))
+
     def formatParams(s):
         for param in params:
-            s = s.replace('%'+param+'%', str(params[param]))
+            s = s.replace('{'+param+'}', str(params[param]))
```

Comment:
Consistency is definitely best. =) The shell supports both `$foo` and `${foo}`; the latter is for exactly the `{latency}ms` type of case. So maybe we should just support both when we do our own formatting? That solves your issue and also keeps things consistent.